wordpress-mobile/WordPress-iOS
# [SIWA] Using methods to create WP account

Fixes #n/a

WPKit PR: https://github.com/wordpress-mobile/WordPressKit-iOS/pull/181
WPAuth PR: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/116

This uses the changes in both pods listed above to attempt WP account creation with Apple ID credentials.

**NOTE**: The endpoint isn't updated yet, so account creation will fail.

To test:
Run the app in Xcode 11.
- Log out if necessary.
- Select Log In > Continue with Apple.
- Walk through the process.
- After your password is entered, since account creation doesn't work yet:
  - The view will be dismissed.
  - A failure error will appear in the console. Ex:
`Apple Authenticator: createWPComUserWithApple failure. error: Error Domain=WordPressKit.WordPressComRestApiError Code=7 "The given service is not valid."`

Run the app in Xcode 10. Verify the app runs and does not show the 'Continue with Apple' login button.

Update release notes:
